docs/api.md

```diff
@@ -41,6 +41,10 @@ mainly, but not only, provisioning details.
       * `irmc://<host>:<port>`, where `<port>` is optional if using the default.
     * HUAWEI ibmc
       * `ibmc://<host>:<port>` (or `ibmc+http://<host>:<port>` to disable TLS)
+    * HPE iLO 4
+      * `ilo4://<host>:<port>` (or `ilo4+http://<host>:<port>` to disable TLS),
```

Comment:
Any plans to support virtual media as well?

Reply:
Yeah, but since ilo driver does not support vmedia deploy using kernel and ramdisk in ironic. We are working on that. Once that gets merged. Then will add the support here.